content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: blog
+title: "Kubernetes 1.17: Stability"
+date: 2019-12-10
+slug: kubernetes-1-17-release-announcement
+---
+
+**Authors:** [Kubernetes 1.17 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.17/release_team.md)
+
+We’re pleased to announce the delivery of Kubernetes 1.17, our fourth and final release of 2019! Kubernetes v1.17 consists of 22 enhancements: 14 enhancements have graduated to stable, 4 enhancements are moving to beta, and 4 enhancements are entering alpha.
+
+# Major Themes
+
```

Comment:
Hello @rawkode .
nit: formatting: Typically, I _think_ the H1 level header is not used.